assets/js/tag-manager.js

```diff
@@ -0,0 +1,15 @@
+dataLayer = [{ activeCaseloadId: '{{ user.activeCaseLoadId }}', userId: '{{ user.username | MD5 | upper }}' }]
```

Comment:
We might have a problem with this actually, I hadn't spotted that it has nunjucks variables passed to it.

One way to resolve this is to have variables in the html like so:

```
<input type="hidden" id="active-caseload" value='{{ user.activeCaseLoadId }}'>
<input type="hidden" id="user-id" value='{{ user.username | MD5 | upper }}'>
```

And then in this javascript file get the variables like so:

```
dataLayer = [{ activeCaseloadId: $('#active-caseload').val(), userId: $('#user-id').val() }]
```

Reply:
Would jquery need to load first for this to work?
Or... add these values to the script server-side with a cache bust in the markup.  At the page level? Might be OK at session level.

Reply:
Ah you're right, looks like this gets loaded before jquery, I suppose it can be done without though,
`document.getElementById("active-caseload").value` etc.